pkg/query-service/app/dashboards/model.go

```diff
@@ -141,6 +148,7 @@ type Dashboard struct {
 	UpdateBy  *string   `json:"updated_by" db:"updated_by"`
 	Title     string    `json:"-" db:"-"`
 	Data      Data      `json:"data" db:"data"`
+	Locked    *int      `json:"locked" db:"locked"`
```

Comment:
doesn't this get exported?

Reply:
No, only the `Data` part gets exported or imported. The JSON from the `data` field is the only part we export or import (Look at the API response of dashboard calls for a better idea).

Reply:
ok